lib/kvdb/kvdb_ctxn.c

```diff
@@ -153,12 +154,17 @@ kvdb_ctxn_reaper(struct work_struct *work)
     cds_list_for_each_entry_rcu(ctxn, &ktn->ktn_alloc_list, ctxn_alloc_link) {
         state = seqnoref_to_state(ctxn->ctxn_seqref);
         if (state == KVDB_CTXN_ACTIVE) {
-            if (now > (ctxn->ctxn_begin_ts + ttl_ns))
+            if (now > (ctxn->ctxn_begin_ts + ttl_ns)) {
                 list_add(&ctxn->ctxn_abort_link, &alist);
+                ++abort_cnt;
+            }
         }
     }
     rcu_read_unlock();
 
+    if (abort_cnt)
+        log_info("Aborting %u transactions (expired)", abort_cnt);
```

Comment:
Is INFO the right level for this log? Would a user find this useful?

Reply:
Seems to depend on whether or not the app is expecting this...